lib/developer_portal/app/views/developer_portal/docs.html.liquid

```diff
@@ -2,20 +2,8 @@
 
 <p>Use our live documentation to learn about the Echo API</p>
 
-{% cdn_asset /swagger-ui/2.2.10/swagger-ui.js %}
-{% cdn_asset /swagger-ui/2.2.10/swagger-ui.css %}
+{% content_for javascripts %}
+    {{ 'active_docs.js' | javascript_include_tag }}
```

Comment:
I'm not sure if importing this way is going to work, the content in `cdn_asset` is supposed to be retrieved from a cdn (our own).
Why are you moving it from the cdn?

Reply:
The main reason is to keep track of the versioning. This way we no longer have to expose the entire bundle and migrate templates but wepacker/rails take care of it. It's supposed to work.. at least it does in local. @didierofrivia might be able to explain further (and better)

Reply:
Yeah, we are also aiming to get rid of modified 3rd party's assets (providing them with the cdn). Instead, having specific 3scale plugins (in the case of swagger specifically)